docs/contents.md

```diff
@@ -42,10 +43,15 @@ When creating notification rules, selecting **Manual Input** as a notification t
 ## Playbook Collection
 
 | 02 - Use Case - FortiSOAR for Microsoft Teams |
-|:------------------------------------|
+|:----------------------------------------------|
+
+| Playbook Name                  | Description                                                                                                                    |
+|:-------------------------------|:-------------------------------------------------------------------------------------------------------------------------------|
+| Create Indicator               | Extract Indicators from the text provided as Input                                                                             |
+| Create Alert                   | Creates an alert based on the values provided in the form                                                                      |
+| Enrich IP From Microsoft Teams | Reference playbook that gets triggered from Microsoft Teams based and provides latest reputation formatted for Microsoft Teams |
```

Comment:
This description should be changed to "Reference playbook that gets triggered and provides latest well formatted reputation using configured threat Intel platform"